go.mod

```diff
@@ -6,8 +6,7 @@ toolchain go1.22.2
 
 require (
 	github.com/container-storage-interface/spec v1.9.0
-	github.com/onsi/ginkgo/v2 v2.17.2
-	github.com/onsi/gomega v1.33.1
+	github.com/digitalocean/go-qemu v0.0.0-20230711162256-2e3d0186973e
 	github.com/openshift/custom-resource-status v1.1.2
```

Comment:
Should go.mod changes be in their own patch (either before or after vendoring)?

Reply:
I think this can be lumped with vendoring since you would typically run `go mod tidy` with the vendoring command to make sure your deps are all squared away